pages/login.tsx

```diff
@@ -107,6 +107,9 @@ const Login: NextPage = () => {
               <Icon
                 name="flanksource-icon"
```

Comment:
```suggestion
                name="flanksource"
```

Reply:
Can we switch to the flanksource logo and not the icon

Reply:
should I use this logo

![Logo](https://uploads-ssl.webflow.com/62c72370a0f3f0f10d12b623/62dfb6160e142e610525ac85_Logo%20Blue.svg)